odxtools/diaglayer.py

```diff
@@ -492,7 +521,8 @@ def get_send_id(self) -> Optional[int]:
 
     def get_can_func_req_id(self, protocol_name: Optional[str] = None) -> Optional[int]:
         """CAN Functional Request Id."""
-        com_param = self.get_communication_parameter("CP_CanFuncReqId", protocol_name=protocol_name)
+        com_param = self.get_communication_parameter(
+            "ISO_15765_2.CP_CanFuncReqId", protocol_name=protocol_name)
```

Comment:
sure about the name here?

Reply:
I'm sure. That it is wrong. thanks for the catch. Fixed: c353954686f7366f825961203c52e41f39cdc2f0...